packages/client/hmi-client/src/components/model/petrinet/tera-petrinet-tables.vue

```diff
@@ -37,7 +37,13 @@
 				Observables <span class="artifact-amount">({{ observables.length }})</span>
 			</template>
 			<DataTable v-if="!isEmpty(observables)" edit-mode="cell" data-key="id" :value="observables">
-				<Column field="id" header="Symbol" />
+				<Column field="id" header="Symbol">
+					<template #body="slotProps">
+						<span class="latin-modern-font">
+							{{ slotProps.data.id }}
+						</span>
+					</template>
+				</Column>
```

Comment:
Can't you add a class to the Column directly?

Reply:
That will change the title